examples/specialty_plots/ishikawa.py

```diff
@@ -0,0 +1,229 @@
+                y_start = y_end - s * np.sin(np.radians(alpha))
+            else:
+                x_end = xpe - s * np.cos(np.radians(palpha))
+                y_end = ype - s * np.sin(np.radians(palpha))
+
+                x_start = x_end - s * np.cos(np.radians(alpha))
+                y_start = y_end - s * np.sin(np.radians(alpha))
+            if debug:
+                _log.debug(
+                    f'k: {k}, x_start: {x_start}, y_start: {y_start}, '
+                    f'x_end: {x_end} y_end: {y_end}, alpha: {alpha} \n '
+                    f'reclevel: {reclevel}, '
+                    f's:{s}, plen:{plen}, palpha:{palpha}')
+
+            # draw arrow arc
+            if reclevel == 1:
+                props = dict(boxstyle='round', facecolor='lightsteelblue', alpha=1.0)
+                spine = ax.annotate(str.upper(k), xy=(x_end, y_end),
+                                    xytext=(x_start, y_start),
+                                    arrowprops=dict(arrowstyle="->",
+                                                    facecolor=spine_color),
+                                    bbox=props, weight='bold')
+            else:
+                props = dict(boxstyle='round', facecolor='lavender', alpha=1.0)
+                spine = ax.annotate(k, xy=(x_end, y_end), xytext=(x_start, y_start),
+                                    arrowprops=dict(arrowstyle="->",
+                                                    facecolor=spine_color),
+                                    bbox=props)
+            # Call recursion to draw subspines
+            drawspines(data=data[k], ax=ax, parentspine=spine,
+                       alpha_ps=alpha_ps, alpha_ss=alpha_ss,
+                       spine_color=spine_color, reclevel=reclevel + 1,
+                       max_recursion_level=max_recursion_level)
+            # no primary_spines_rel_xpos is needed to be passed on recursion
+            # next spine settings - same level
+            alpha *= -1
+            spine_count += 1
+            if spine_count % 2 == 0:
+                s = s + spacing
+    return None
+
+
+def ishikawaplot(data: dict, ax: plt.Axes, left_margin: float = 0.05,
+                 right_margin: float = 0.05, alpha_ps: float = 60.0,
+                 alpha_ss=0.0, primary_spines_rel_xpos: List[float] = [np.nan],
+                 pd_width: int = 0.1, spine_color: str = "black") -> None:
+    """
+
+    Parameters
+    ----------
+    data : dict
+        Plot data structure.
+    ax : matplotlib.pyplot.Axes
+        Axes in which to drow the plot.
+    left_margin : float, optional
+        Left spacing from frame border. The default is 0.05.
+    right_margin : float, optional
+        Right spacing from frame border. The default is 0.05.
+    alpha_ps : float, optional
+        First spine angle using during recursion. The default is 60.0.
+    alpha_ss : TYPE, optional
+        Secondary spine angle using during recursion. The default is 0.0.
```

Comment:
Also rename as above
```suggestion
    primary_spine_angle : float, optional
        First spine angle using during recursion. The default is 60.0.
    secondary_spine_angle : float, optional
        Secondary spine angle using during recursion. The default is 0.0.
```